### PR TITLE
fixed ios build and crash at startup

### PR DESCRIPTION
**Pull request by asm-enthusiast**
I fixed the iOs build and a startup crash.

For the build, apparently apple changed what is acceptable for PRODUCT_BUNDLE_IDENTIFIER and in net.e1547 "net." is invalid, so it got changed to clragon.e1547.
For the crash, apparently these
```
<key>NSAppTransportSecurity</key>
<key>NSAllowsArbitraryLoads</key>
```
are not valid Info.plist keys and would cause a crash at app startup.

I also applied recommendation from XCode 12 which set lower API level to iOs12.

it has been tested on simulator and iPadPro 3rd gen.

### CODE REVIEW

File: ios/Runner/AppDelegate.swift [hunk 1]
```diff
@@ -7,10 +7,8 @@ import Flutter
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
-    var flutter_native_splash = 1
```
Comment (clragon) [inline ios/Runner/AppDelegate.swift:10]: does removing this not break the splash screen?

These lines are inserted by a plugin that displays a native splash screen.

**Review by clragon — DISMISSED** [review]

**Comment by clragon** [comment]
looks very good, thank you for your work.

so, `net.e1547` is not a valid identifier? very odd.

I will merge this as soon as the splash screen has been resolved.

File: ios/Runner/AppDelegate.swift (hunk 1 above)
Comment (asm-enthusiast) [reply ios/Runner/AppDelegate.swift:10]: I forgot to mention that XCode marked this attribute as deprecated that's why it got removed, I did not notice anything wrong with the splash screen. I'll check again in the evening Paris time and confirm that it works fine.

Comment (asm-enthusiast) [reply ios/Runner/AppDelegate.swift:10]: So, the actual message is
Screenshot 2020-11-13 at 18 10 11

and here is a recording of running the app without it:

e1547-splash-record

As for the net. issue, I created a new empty flutter project as a test and they default to a net. identifier except xcode 12 rejects it.

**Pull request merged**